packages/design-carousel/src/item.tsx

```diff
@@ -28,7 +28,7 @@ export function Item( { style, design, className, type }: Props ) {
 	let options: MShotsOptions = { w: 400, vpw: 400, vph: 872, format: 'png' };
 
 	if ( type === 'desktop' ) {
-		options = { w: 1280, vpw: 1920, vph: 1280, format: 'png' };
+		options = { w: 1920, vpw: 1920, vph: 1280, format: 'png' };
```

Comment:
@sixhours, might it be worth adding a media query check for this logic? Something like the following might work?
```suggestion
import { useMediaQuery } from '@wordpress/compose';
...
		// This const likely needs to be set higher up due to us using a hook, but including it here for illustrative purposes
		const isLargerThan1440px = useMediaQuery( '(min-width: 1440px)' );
		options = {
			w: isLargerThan1440px ? 1920 : 1280,
			vpw: 1920,
			vph: 1280,
			format: 'png',
		};
```

Reply:
TIL about `useMediaQuery`! I'll take a look at this!